### PR TITLE
Add v1.8.1 of mattermost-plugin-gitlab to the Marketplace

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
#### Ticket Link
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/460

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2,8 +2,157 @@
...
+          }
+        ]
+      }
+    },
+    "platforms": {
+      "linux-amd64": {
+        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-gitlab-v1.8.1-linux-amd64.tar.gz",
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmXU70oACgkQ0bVLR6XO/sSrFQ/+J4EgmVgtCCDNmnSu3ep725qXLS+3GMmQKmAYrxFpxBTjXyhfdrw7HHR7QVk8U6zRIwP45zkxFHjoq6jC9KtfYT8CE62scpxc7jUhIercLdlwPrsAPocjHkCwy197egh3tWO/h1tEYCyHQ4N/YSmBKisJX/SgxRpQIQ1PmlWyRvvkN/7+VUzvsnVVzP+srrmuMoqXsR3YffBCFp3Rae4doLtrLPD7NEHI7FoDFqorad91N0h5wdSF81jp9aA3ClFCONBu4MODO/nU3zMXKJ6xhajMfXuSFYP1twopnjcRQAJOfA20ZYfqEO5Y5PX+nsep4ysYuMNhvcHLmu8lV26ZR1UsRbgp66DEFnuWrd+10dXw6huYouFML2gsFznatYI4NuwbOsZU65OuQMbQycac3y+qdZ7rr22zQzfJOLMCsM5WbNUh/oBLmE6www8kZJBZQuzqbg4o/8xKNXYOK00w8D0BKdrMmSCF+tDWh0wDOEfUREpDYgS1xsMcso4FlJzsopxuOdUREx+qcBINWli2K0Y3rmGu9y/lZzUytMiMz9SKlhkM6mXs5LL19ByIAOKFuykkzlb8KCP9hP1gX1Z5TYVow/Cn5jSdp9kDhskiw6Cmd7UNzvslr8cycl4bVAnPyx3cIfeA30+aIu0nItVKDUB0P6BSCcMjvPWCg/9sAIA="
+      },
+      "darwin-amd64": {},
```
Comment (mickmister) [inline plugins.json:143]: I think we can continue making marketplace releases with this issue, but we are blocked on prepackage PR CI runs with this right now. We will likely need to re-cut the releases, or skip these particular releases for prepackaging.

**Review by mickmister — APPROVED** [review]

File: plugins.json [hunk 2]
```diff
@@ -16,9 +165,9 @@
       "description": "GitLab plugin for Mattermost.",
       "homepage_url": "https://github.com/mattermost/mattermost-plugin-gitlab",
       "support_url": "https://github.com/mattermost/mattermost-plugin-gitlab/issues",
-      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.1",
```
Comment (mickmister) [inline plugins.json:168]: I think these changes should be reverted, to keep the `1.8.0` release entry in tact

**Review by mickmister — CHANGES_REQUESTED** [review]

**Review by hanzei — APPROVED** [review]

File: plugins.json (hunk 2 above)
Comment (raghavaggarwal2308) [reply plugins.json]: @mickmister Reverted

**Review by hanzei — APPROVED** [review]
Nice one :+1:

**Review by mickmister — APPROVED** [review]

**Pull request merged**